ion-transformation-storage-api/src/main/java/com/connexta/transformation/storage/api/StorageProvider.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.storage.api;
+
+public interface StorageProvider {
```

Comment:
❗️ We need a `create` operation that would be used to prep the storage/database for a given piece of intel to be transformed. This operation would need to expect the URIs were are receiving as part of the transform request and the pre-existing metadata (a list of) and would return a transform id corresponding to the newly created transform request.